### ISSUE TITLE
Some scripted NPC speech broken

### ISSUE THREAD

**Issue by Nazxul360** [LAGonauta/MetaAudio#16]
MetaAudio Oct 13 2019 Release
OpenAL 1.1
Half-Life version current at time of posting
Launch parameters: metahook.exe -steam -game valve -insecure

How to reproduce:
1. Use "map c3a2d" to load last map of Lambda Core
2. Use "give item_suit" and "impulse 101" to avoid potential bugs caused by not having items.
3. Proceed to the lab entrance where the scientist opens the door and begins talking.
4. When he says "I'm sorry to say, but you'll have to kill it" all dialogue stops, even though the conversation is supposed to continue. The NPCs mouths don't even move. Everything else in the game proceeds normally, and you can get the guard to follow you or wait, but he won't say anything. The scientist in the portal chamber won't speak either. In fact no NPC in the game will say a word, even if you load a different save. The only way to get them talking again is to close and restart the game. I also had this happen in a few other instances, such as in Questionable Ethics when you get to the back entrance of the medical lab with the three scientists, but have been unable to reproduce it.

**Comment by LAGonauta** [comment]
Seems to work fine when running in Debug mode, but not in Release mode.

**Comment by Nazxul360** [comment]
How do I start in debug mode? I'd like to give it a try.

Thanks!

**Comment by LAGonauta** [comment]
You would have to install Visual Studio and compile it yourself as it needs some libraries in debug mode to work :/

**Comment by Nazxul360** [comment]
Oof. I just briefly looked into that and it seems way over my head. I have no programming/building experience and don't have the time nor the interest to research and learn.

If you have the time and are willing to tell me exactly what I need to do step by step, I would really appreciate it. Otherwise I guess I'll just wait for a release where this has been fixed. ;)

Looking forward to a fix - your work on this is incredible and I can't wait to use it! Thanks again.

**Comment by Nazxul360** [comment]
Hi again,

If it helps you at all, I've pinpointed another instance where this bug occurs. In Blue Shift, right after you return from Xen and Dr. Rosenberg is explaining that the power cell has been drained, as soon as he says "someone will have to go down to the lower level and retrieve another power cell," all dialogue stops again.

Thanks again for your work!

**Comment by LAGonauta** [comment]
@Nazxul360 Can you please test?

[MetaAudio.Alure.zip](https://github.com/LAGonauta/MetaAudio/files/4050716/MetaAudio.Alure.zip)

**Comment by Nazxul360** [comment]
Thanks, will do! I'll try to check it out within the next 24 hours.

**Comment by Nazxul360** [comment]
@LAGonauta So I managed to do a quick test this morning in the first scenario mentioned in my original post, and it looks like the issue has been resolved!

I got an error at first saying that it couldn't find "audio.dll". I noticed that the .dll in metahook/plugins/ had been changed to "MetaAudio.dll" so I just renamed it back to "audio.dll" and it all worked fine after that.

Hopefully I will have time to check the Blue Shift scene and play around with it some more tonight after work. I'll let you know how I go, but it all looks (sounds) good so far! Is there anything specific you would like me to try?

Can't thank you enough!
